tokenExchange/src/app/publicKeyGetter.js

```diff
@@ -0,0 +1,64 @@
+const jwkToPem = require('jwk-to-pem');
+const axios = require('axios');
+let cachedKeyPem = {};
+
+module.exports = {
+    async getPublicKey ( issuer, kid ){
+        let keyInPemFormat
+        if ( cachedKeyPem && cachedKeyPem.expiresOn > Date.now() ) {
+            console.debug( 'Using cached key pem' )
+            keyInPemFormat = cachedKeyPem.value
+        } else {
+            jwksendpoint = getJwksEndpoint( issuer )
+            let jwKey = await getJwkByKid( jwksendpoint, kid );
+            console.debug('get jwkey ok');
+            keyInPemFormat = jwkToPem(jwKey);
+            console.debug('get key in pem format ok ');
+            setCachedData( keyInPemFormat )
+        } 
+        return keyInPemFormat;
+    }
+}
+
+const issuersUrl = {
+    'api.selfcare.pagopa.it': 'https://uat.selfcare.pagopa.it/.well-known/jwks.json', //TODO vedi stato issue SELC-390
+    'spidhub-test.dev.pn.pagopa.it':'http://spidhub-test.dev.pn.pagopa.it:9090/.well-known/jwks.json'
+}
```

Comment:
Potrebbe essere utile caricarlo da variabile d'ambiente in formato JSON. Se la variabile d'ambiente ha un valore non vuoto allora si fa un JSON.parse( <Variabile d'ambiente> ) altrimenti si usano i valori clabalti.